academic-services/knowledge-exploration-service/tutorial-entity-linking.md

```diff
@@ -144,7 +144,7 @@ If we're able to find an interpretation that meets our confidence cut off score,
 
 The sample entity linking script output should look like the following:
```

Comment:
>look like the following [](start = 47, length = 23)

"look similar to the following (confidence scores may change with newer data releases):"